content/en/blog/2022/get-started-ambient/index.md

```diff
@@ -6,6 +6,10 @@ attribution: "Lin Sun (Solo.io), John Howard (Google)"
 keywords: [ambient,demo,guide]
 ---
 
+{{< warning >}}
+Refer to the latest [getting started with ambient mesh doc](http://preliminary.istio.io/latest/docs/ops/ambient/getting-started/) for updated instructions.
```

Comment:
I need to verify, but does making this link /docs... mean that it will point to preliminary.istio.io here, and when we branch for 1.18, it's point to the 1.18 version. I think that's what we want.

Unless we are planning to cherry-pick to the current istio.io, where we want to point back to the preliminary.istio for now.

Reply:
> Unless we are planning to cherry-pick to the current istio.io, where we want to point back to the preliminary.istio for now.

Yes, that's what we want.

Reply:
yep, so we don't need to keep cherry pick this doc, which could undergo quite some revisions!